index.js

```diff
@@ -1,10 +1,16 @@
-import {NativeModules, DeviceEventEmitter} from 'react-native';
+import {NativeModules, DeviceEventEmitter, AppState, Platform, PushNotificationIOS} from 'react-native';
 
 const eventsMap = {
     refreshToken: 'FCMTokenRefreshed',
     notification: 'FCMNotificationReceived'
 };
 
+let currentAppState = 'active';
```

Comment:
from the logic in the code, you are treating app state as `active` if the state is empty. once the state is updated, it is the same as AppState. 
I prefer to put the logic in notification data handlers because it is just javascript utilities and people can decide what they want to do. (either treat null as active or wait until state is updated etc)